Major Update (#268)
NilPointer-Software/bdfd-wiki: src/bdscript/removeContains.md

# $removeContains

> Removes messages that contains provided words. It will remove `amount` last messages (replace 'amount' with number) (max 100 messages) 

Reviewer: ```suggestion
Removes messages that contains provided words. Removes up to the given `amount` of latest messages (max 100) 
```

## Usage
```

> $removeContains[word;...;Amount]

Reviewer: ```suggestion
$removeContains[word;...;amount]
```

```

### Parameters
- `word` `(Type : String || Flag : Emptiable)` : The phrases/words to delete, separate phrases using `;`.

> - `Amount` `(Type : Integer || Flag : Required)` : How many messages containing the provided phrases to delete. Max is 100.

Reviewer: ```suggestion
- `word` `(Type: String || Flag: Emptiable)`: The phrases/words to delete, separate phrases using `;`.
- `amount` `(Type: Integer || Flag: Required)`: The number of messages containing given words to delete. Max is 100.
```
